pkg/constants/k8s_constants.go

```diff
@@ -50,6 +50,17 @@ const (
 	CNIFlannelVersion   = "0.22.3"
 	CNIFlannelImageTag  = "v0.22.3-2"
 
+	KubeProxyRelease   = "kube-proxy"
+	KubeProxyNamespace = "kube-system"
+	KubeProxyChart     = "kube-proxy"
+	KubeProxyVersion   = "2.0.0"
+	KubeProxyDaemonSet = "kube-proxy"
+	KubeProxyConfigMap = "kube-proxy"
+	KubeProxyConfigMapConfig = "config.conf"
+	KubeProxyConfigMapKubeconfig = "kubeconfig.conf"
+	KubeProxyImage     = "container-registry.oracle.com/olcne/kube-proxy"
```

Comment:
should the repo name be hard coded?

Reply:
Yes, because the images are pre-pulled into OCK like with the other necessary images such as kube-apiserver, etcd, and so on.